README.md

```diff
@@ -14,7 +14,7 @@ Crucible should only need to be installed on a single Linux system (the crucible
 
 ### Hybrid/Multi-cloud Capable
 
-Crucible is designed to not just support different cloud solutions, but support testing multiple clouds at the same time, and even mixed cloud solutions in the same test. Crucible implements what are called "endpoints", and each endpoint type facilitates benchmark and tool execution on a specific type of system, cluster, or cloud. The most basic endpoint is "localhost" (benchmark/tools on the crucible-controller host), and as of this writing, there is also "remotehost" (benchmark/tools on remote host reachable via ssh) and "k8s" (a kubernetes cluster). These endpoints can be used for the same crucible-run (a set of benchmark tests), for example, a remotehost endpoint can be used with a k8s endpoint, where the remotehost endpoint is a benchmark-client and the benchmark-server is running in a kubernetes cluster. Other endpoint types are planed, like "ovirt" for testing [oVirt](https://www.ovirt.org) managed KVM clusters, "kvmhost" for testing KVM VMs from a single host via [libvirt](https://libvirt.org), and endpoints for clouds services like [ec2](https://aws.amazon.com/ec2/), [cge](https://cloud.google.com/computeGCE), and [Azure](https://azure.microsoft.com).
+Crucible is designed to not just support different cloud solutions, but support testing multiple clouds at the same time, and even mixed cloud solutions in the same test. Crucible implements what are called "endpoints", and each endpoint type facilitates benchmark and tool execution on a specific type of system, cluster, or cloud. The most basic endpoint is "localhost" (benchmark/tools on the crucible-controller host), and as of this writing, there is also "remotehost" (benchmark/tools on remote host reachable via ssh) and "k8s" (a kubernetes cluster). These endpoints can be used for the same crucible-run (a set of benchmark tests), for example, a remotehost endpoint can be used with a k8s endpoint, where the remotehost endpoint is a benchmark-client and the benchmark-server is running in a kubernetes cluster. Other endpoint types are planned, like "ovirt" for testing [oVirt](https://www.ovirt.org) managed KVM clusters, "kvmhost" for testing KVM VMs from a single host via [libvirt](https://libvirt.org), and endpoints for clouds services like [ec2](https://aws.amazon.com/ec2/), [cge](https://cloud.google.com/computeGCE), and [Azure](https://azure.microsoft.com).
```

Comment:
We should remove information about the "localhost" endpoint and add the "OSP" endpoint to the list.

Reply:
I've created a new commit with the requested changes.